Panic creating file

```
panic: CreateFile C:\Users\Alex\.samp\packages\YSI-Internal\y_core\default: Access is denied.

goroutine 1 [running]:
github.com/Southclaws/sampctl/util.Exists(0xc0427da400, 0x38, 0x5)
        /home/travis/gopath/src/github.com/Southclaws/sampctl/util/util.go:94 +0xae
github.com/Southclaws/sampctl/rook.PackageContext.ensureRepoExists(0x1, 0xc0422bb880, 0x37, 0xc04201fd10, 0x44, 0xc04201e67d, 0x4, 0x0, 0x0, 0xc0423a75b0, ...)
        /home/travis/gopath/src/github.com/Southclaws/sampctl/rook/cache.go:184 +0x1b3
github.com/Southclaws/sampctl/rook.PackageContext.EnsureDependencyCached(0x1, 0xc0422bb880, 0x37, 0xc04201fd10, 0x44, 0xc04201e67d, 0x4, 0x0, 0x0, 0xc0423a75b0, ...)
        /home/travis/gopath/src/github.com/Southclaws/sampctl/rook/cache.go:177 +0x15f
github.com/Southclaws/sampctl/rook.(*PackageContext).EnsureDependenciesCached.func1(0xb76e2d, 0xa, 0xc0429fd720, 0xc, 0xc0429fd72d, 0x6, 0xc0429fd720, 0x0, 0x0, 0x0, ...)
        /home/travis/gopath/src/github.com/Southclaws/sampctl/rook/cache.go:67 +0x148a
github.com/Southclaws/sampctl/rook.(*PackageContext).EnsureDependenciesCached.func1(0xb76e2d, 0xa, 0xc0423a75f0, 0x8, 0xc0423a75f9, 0x7, 0xc0423a75f0, 0x0, 0x0, 0x0, ...)
        /home/travis/gopath/src/github.com/Southclaws/sampctl/rook/cache.go:143 +0x1090
github.com/Southclaws/sampctl/rook.(*PackageContext).EnsureDependenciesCached.func1(0x0, 0x0, 0xc0423a75b0, 0xa, 0xc0423a75c0, 0xb, 0x0, 0x0, 0x0, 0x0, ...)
        /home/travis/gopath/src/github.com/Southclaws/sampctl/rook/cache.go:143 +0x1090
github.com/Southclaws/sampctl/rook.(*PackageContext).EnsureDependenciesCached(0xc042015600, 0x2, 0x2)
        /home/travis/gopath/src/github.com/Southclaws/sampctl/rook/cache.go:150 +0x278
github.com/Southclaws/sampctl/rook.NewPackageContext(0xc04209e6c0, 0x42b03d8, 0xc0421d29c0, 0x1, 0xc0422bb880, 0x37, 0xb74f43, 0x7, 0xc04243b0c0, 0x13, ...)
        /home/travis/gopath/src/github.com/Southclaws/sampctl/rook/load.go:106 +0x4c3
main.packageInstall(0xc0420c0580, 0x0, 0xc0420c0580)
        /home/travis/gopath/src/github.com/Southclaws/sampctl/package_install.go:65 +0x320
github.com/Southclaws/sampctl/vendor/gopkg.in/urfave/cli%2ev1.HandleAction(0xa633a0, 0xba1a80, 0xc0420c0580, 0xc0423ab500, 0x0)
        /home/travis/gopath/src/github.com/Southclaws/sampctl/vendor/gopkg.in/urfave/cli.v1/app.go:490 +0xcf
github.com/Southclaws/sampctl/vendor/gopkg.in/urfave/cli%2ev1.Command.Run(0xb74786, 0x7, 0x0, 0x0, 0x0, 0x0, 0x0, 0xb912fa, 0x2c, 0x0, ...)
        /home/travis/gopath/src/github.com/Southclaws/sampctl/vendor/gopkg.in/urfave/cli.v1/command.go:210 +0xa3d
github.com/Southclaws/sampctl/vendor/gopkg.in/urfave/cli%2ev1.(*App).RunAsSubcommand(0xc0420a5040, 0xc0420c0160, 0x0, 0x0)
        /home/travis/gopath/src/github.com/Southclaws/sampctl/vendor/gopkg.in/urfave/cli.v1/app.go:379 +0xa17
github.com/Southclaws/sampctl/vendor/gopkg.in/urfave/cli%2ev1.Command.startApp(0xb74ad5, 0x7, 0x0, 0x0, 0xc042562360, 0x1, 0x1, 0xb85bbc, 0x1c, 0x0, ...)
        /home/travis/gopath/src/github.com/Southclaws/sampctl/vendor/gopkg.in/urfave/cli.v1/command.go:298 +0x8e0
github.com/Southclaws/sampctl/vendor/gopkg.in/urfave/cli%2ev1.Command.Run(0xb74ad5, 0x7, 0x0, 0x0, 0xc042562360, 0x1, 0x1, 0xb85bbc, 0x1c, 0x0, ...)
        /home/travis/gopath/src/github.com/Southclaws/sampctl/vendor/gopkg.in/urfave/cli.v1/command.go:98 +0x124d
github.com/Southclaws/sampctl/vendor/gopkg.in/urfave/cli%2ev1.(*App).Run(0xc0420a41a0, 0xc042090000, 0x5, 0x8, 0x0, 0x0)
        /home/travis/gopath/src/github.com/Southclaws/sampctl/vendor/gopkg.in/urfave/cli.v1/app.go:255 +0x6a7
main.main()
        /home/travis/gopath/src/github.com/Southclaws/sampctl/main.go:301 +0x1986
YSI-Internal/y_core
```

The strange thing is, this was in the cache, and I know the file existed before because I was ensuing a long string of projects.  It seemed to work for a while, then this happened.

--- a/pkgcontext/cache.go
+++ b/pkgcontext/cache.go
@@ -197,7 +197,8 @@ func (pcx PackageContext) ensureRepoExists(
 	repo, err = git.PlainOpen(to)
 	if err != nil {
 		print.Verb("no repo at", to, "-", err, "cloning new copy")
-		if util.Exists(to) {
+
+		if _, err = os.Stat(to); !os.IsNotExist(err) {
 			print.Verb("removing existing folder", to)
 			err = os.RemoveAll(to)
 			if err != nil {

